examples/webhook-signing/nestjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "webhook-signing-example-nestjs",
+  "version": "1.0.0",
+  "description": "Nestjs webhook parsing sample",
+  "repository": {},
+  "main": "./main.ts",
+  "scripts": {
+    "run": "ts-node-transpile-only ./main.ts",
+    "prepare": "../prepare.sh"
+  },
+  "author": "Ali karimi",
+  "license": "ISC",
+  "dependencies": {
+    "@nestjs/common": "^10.2.1",
+    "@nestjs/config": "^3.0.0",
+    "@nestjs/core": "^10.2.1",
+    "dotenv": "^16.3.1",
+    "@nestjs/platform-express": "^10.2.1",
+    "@sggc/stripe-nestjs": "^1.0.7",
```

Comment:
We don't mind referencing popular and supported community libraries but sggc/stripe-nestjs is quite new and not widely adopted. 

Is it possible to reduce the sample to avoid external dependencies other than the `nestjs`?

Reply:
sggc/stripe-nestjs  removed